src/ui/public/typeahead/partials/typeahead-items.html

```diff
@@ -5,7 +5,10 @@
   class="typeahead-items"
   >
 
+  <div ng-if="!typeahead.query.length" ng-transclude></div>
```

Comment:
I played around with that idea before, I was afraid it might be too noisy. I'm down with it if you think it looks ok though.